src/components/Detail/TabRecipes.tsx

```diff
@@ -0,0 +1,23 @@
+import { MetadataTab } from "./MetadataTab";
+import { MiradorTab } from "./MiradorTab";
+import { TextComponentTab } from "./TextComponentTab";
+import { WebAnnoTab } from "./WebAnnoTab";
+
+export const TabRecipes = {
```

Comment:
De term Recipe verwart misschien een beetje? Er wordt niet bereid, en er is ook niet echt een stappenplan of zo. Maar ja, wat is dan een betere term.. Misschien een TabTemplate? Of wat zit er in een Tab, een TabDocument, of TabComponent?